SPECS/dbus-glib/dbus-glib.spec

```diff
@@ -1,49 +1,52 @@
-Summary:	Glib interfaces to D-Bus API
-Name:		dbus-glib
-Version:	0.110
-Release:        3%{?dist}
-License: 	AFL and GPLv2+
-Group: 		System Environment/Libraries
-Source0:	http://dbus.freedesktop.org/releases/dbus-glib/%{name}-%{version}.tar.gz
-%define sha1 dbus-glib=998b7c762c8f18c906f19fc393bb8712eabe8c97
+Summary:        Glib interfaces to D-Bus API
+Name:           dbus-glib
+Version:        0.110
+Release:        4%{?dist}
+License:        AFL OR GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-BuildRequires:	glib-devel
-BuildRequires:	dbus-devel
-Requires:	glib
-Requires:	dbus
-Provides:	pkgconfig(dbus-glib-1)
+Group:          System Environment/Libraries
+URL:            https://dbus.freedesktop.org/doc/dbus-glib/
+Source0:        https://dbus.freedesktop.org/releases/dbus-glib/%{name}-%{version}.tar.gz
+BuildRequires:  dbus-devel
+BuildRequires:  glib-devel
+Requires:       dbus
+Requires:       glib
 
 %description
 The D-Bus GLib package contains GLib interfaces to the D-Bus API.
 
-%package devel
-Summary:	Libraries and headers for the D-Bus GLib bindings
-Requires:	glib-devel
-Requires:	dbus-devel
-Requires:	%{name} = %{version}
+%package        devel
+Summary:        Libraries and headers for the D-Bus GLib bindings
+Requires:       %{name} = %{version}
+Requires:       dbus-devel
+Requires:       glib-devel
+Provides:       pkgconfig(dbus-glib-1)
 
 %description devel
 Headers and static libraries for the D-Bus GLib bindings
 
 %prep
-%setup -q
+%autosetup
+
 %build
-./configure \
-	--prefix=%{_prefix} \
-	--sysconfdir=%{_sysconfdir} \
-        --disable-static \
-	--disable-gtk-doc
+%configure \
+    %if %{with_check}
```

Comment:
I'd remove the whitespaces in front of `%if` and `%endif`. Not sure right now, but I think there were issues with that (or maybe it was in the `Makefile`?). Anyway, we always started from the beginning of the line in other specs, so that's why I'd be careful here.

Reply:
Ran a build both ways, it builds fine as is. Either way, I'll change it to match other specs.